lib/public.js

```diff
@@ -13,7 +13,7 @@ const
     TRACK = 'track',
     TRACK_ONCE_KEY = '__track_once__',
     WRITE_ACCESS_ERROR = 'This value is not writeable.',
-    ERROR_MESSAGE_ID_AND_EMAIL = 'You must identify a user by email or ID.';
+    ERROR_MESSAGE_ID_AND_EMAIL = 'You must identify a user by email or ID.'
```

Comment:
I think we need the semi-colon back on this line after removing the package version import